Tests/SWBTaskConstructionTests/ClangTests.swift

```diff
@@ -142,7 +142,7 @@ fileprivate struct ClangTests: CoreBasedTests {
         }
     }
 
-    @Test(.skipHostOS(.windows, "clang-cache is not available on Windows"), .requireSDKs(.host))
+    @Test(.skipHostOS(.windows, "clang-cache is not available on Windows"), .skipHostOS(.linux, "test is currently failing on linux"), .requireSDKs(.host))
```

Comment:
"requires additional work to support" -- what's the work? could the title include a brief summary of what the issue is?

Reply:
it's somehow incompatible with the fallback system toolchain mechanism. I haven't fully debugged yet